src/pages/home.html

```diff
@@ -0,0 +1,1926 @@
+                                                fill="currentColor"
+                                            />
+                                        </svg>
+                                    </p>
+                                    <div
+                                        class="flex items-center justify-center gap-[6px]"
+                                    >
+                                        <svg
+                                            class="h-4.5 w-4.5"
+                                            viewBox="0 0 27 27"
+                                            fill="none"
+                                            xmlns="http://www.w3.org/2000/svg"
+                                        >
+                                            <path
+                                                d="M7.5 3.375L4.5 7.425V21.6C4.5 22.1371 4.71071 22.6521 5.08579 23.0319C5.46086 23.4117 5.96957 23.625 6.5 23.625H20.5C21.0304 23.625 21.5391 23.4117 21.9142 23.0319C22.2893 22.6521 22.5 22.1371 22.5 21.6V7.425L19.5 3.375H7.5Z"
+                                                stroke="#626E94"
+                                                stroke-width="2"
+                                                stroke-linecap="round"
+                                                stroke-linejoin="round"
+                                            />
+                                            <path
+                                                d="M4.5 7.875H22.5"
+                                                stroke="#626E94"
+                                                stroke-width="2"
+                                                stroke-linecap="round"
+                                                stroke-linejoin="round"
+                                            />
+                                            <path
+                                                d="M18 12.375C18 13.5685 17.5259 14.7131 16.682 15.557C15.8381 16.4009 14.6935 16.875 13.5 16.875C12.3065 16.875 11.1619 16.4009 10.318 15.557C9.47411 14.7131 9 13.5685 9 12.375"
+                                                stroke="#626E94"
+                                                stroke-width="2"
+                                                stroke-linecap="round"
+                                                stroke-linejoin="round"
+                                            />
+                                        </svg>
+                                        <p class="text-sm text-lynch">
+                                            خرید بسته شگفتانه
+                                        </p>
```

Comment:
It is supposed to be an action button or a link. Or the whole box should be linked.